docs/mkdocs/src/platforms/android.md

```diff
@@ -121,3 +120,41 @@ Use the `--format snoop` option to specify that the file is in that specific for
     ```shell
     $ bumble-show --format snoop btsnoop_hci.log
     ```
+
+## Application
+
+Despite there are some sample Applications in this repo, Bumble doesn't have any support to build Bluetooth applications on Android. You may refer to the [Official Doc](https://developer.android.com/develop/connectivity/bluetooth) or other blog articles to learn more about Android and Bluetooth applications.
+
+## Debug
+
+Note that HCI Bridge just serves a bridge between Android Emulator and physical controllers, so if you encounter issues such as `Android cannot scan / pair / connect my another device`, these are usually related to the Android or physical controllers, and not to the Bumble framework.
```

Comment:
"Note that HCI Bridge just serves a bridge between Android Emulator and..." -> "Note that the HCI Bridge just serves a bridge between the Android Emulator and..."